apps/web/pages/booking/[uid].tsx

```diff
@@ -1115,6 +1127,20 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
     });
   }
 
+  const payment = await prisma.payment.findFirst({
+    where: {
+      bookingId: bookingInfo.id,
+    },
+    select: {
+      success: true,
+      refunded: true,
+    },
+  });
+  const paymentStatus = {
+    success: payment?.success,
+    refunded: payment?.refunded,
+  };
+
```

Comment:
Fetching payment but don't sending it via props.

Reply:
It is being sent via props in paymentStatus. I don't see the difference.